pkgs/by-name/pa/passt/package.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchgit, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "passt";
+  version = "0.2023_10_04";
+  src = fetchgit {
+    url = "https://passt.top/passt";
+    rev = "f851084c96e0a932e51c15ad9551e2cd85362147";
+  };
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [];
+  installPhase = ''
```

Comment:
The installPhase should include `runHook preInstall` at the start and `runHook postInstall` at the end.

Reply:
In particular, the `runHook postInstall` should not be optional :)